docs/medical-api/api-reference/document/post-upload-url.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: "Post Upload URL"
```

Comment:
Also, now that we have this endpoint, along with the POST Consolidated, let's add a step `9` to the MAPI quickstart guide titled "Share medical data back with the networks" or something - it's important devs know their options to do this at a glance for shareback requirements